robottelo/vm.py

```diff
@@ -249,6 +249,9 @@ def install_katello_agent(self):
             raise VirtualMachineError('Failed to install katello-agent')
         gofer_check = self.run('service goferd status')
         if gofer_check.return_code != 0:
+            self.run('service goferd start')
```

Comment:
perhaps we should add a function parameter to try to start goferd service with False default value, to not break the current tests and in capsule helper call this function with true,

```python
    def install_katello_agent(self, start_goferd=False):
        ...
        if gofer_check.return_code != 0 and start_goferd:
            self.run('service goferd start')
        ...


```